hedyorg/hedy
# 💻 Stop aproving teacher accounts

Changes the process of becoming a teacher by skipping the approving step. Now users can simply make a teacher account by choosing the option when creating an account or simply choosing to do so from the profile menu.

Fixes #5724

**How to test**

* Create a new account choosing the teacher option. You can use this function to make your life easier:
```
function fill_fields() {
    let username = `teacher_${Math.random()}`;
    $('#username').val(username)
    $('#email').val(username + '@p.ai');
    $('#password').val('123456')
    $('#password_repeat').val('123456')
    $('#language').val('en')
    $('#birth_year').val('1999')
    $('#gender').val('f')
    $('#country').val('VE')    
}
```
* After creating the account you should be able to create classes and such.

Then, you can log into a normal account, go to the profile page and click this button:
![image](https://github.com/user-attachments/assets/5f5d73fe-d571-4f9a-94de-a2bce046147d)

Now your account should be a teacher account.
